compact.js

```diff
@@ -0,0 +1,5 @@
+export function compact(arr){
+ return arr.filter(function(item){
+	if(!!item) return item;
```

Comment:
* nie przeformatowany kod
* zawsze przy ifach nawiasy klamrowe!

można to zrobić następująco:
```javascript
function compact(collection) {
    return item;
}
```